src/Terminal/JupyterLabAppFactory.ts

```diff
@@ -24,6 +24,10 @@ export class JupyterLabAppFactory {
     this.isShellStarted = content?.includes("Connected to") && content?.includes("cqlsh");
   }
 
+  private isPostgresShellStarted(content: string | undefined) {
+    this.isShellStarted = content?.includes("cqlsh");
```

Comment:
Is this check enough to determine whether the shell is Postgres? The Cassandra shell is using the same check too:
`
private isCassandraShellStarted(content: string | undefined) {
    this.isShellStarted = content?.includes("Connected to") && content?.includes("cqlsh");
 }`

Reply:
Good catch! I messed this after a merge.